### PR TITLE
[devtools] Include a helpful DevTools Tip video

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: site/en/docs/devtools/open/index.md [hunk 1]
```diff
@@ -61,3 +62,10 @@ Chrome.
 
 [1]: /docs/devtools/css
 [2]: /docs/devtools/console
+
+
+## What's next?
+
+Congratulations! You've successfully unlocked the power of Chrome DevTools. Next, watch the following video to learn some useful shortcuts and settings for quicker DevTools navigation.
```
Comment (jecfish) [inline site/en/docs/devtools/open/index.md:69]: Not sure "unlock the power" is a good phrase to use here.

Comment (sofiayem) [reply site/en/docs/devtools/open/index.md:69]: I think it's fine.

File: site/en/docs/devtools/open/index.md [hunk 2]
```diff
@@ -61,3 +62,10 @@ Chrome.
...
+
+{% YouTube id="H0XScE08hy8" %}
```
Comment (sofiayem) [inline site/en/docs/devtools/open/index.md:71]: Is that the right video? It's about debugging, not shortcuts and settings.

**Review by sofiayem — CHANGES_REQUESTED** [review]

File: site/en/docs/devtools/open/index.md (hunk 2 above)
Comment (jecfish) [inline site/en/docs/devtools/open/index.md:71]:
```suggestion
{% YouTube id="xHusjrb_34A" %}
```

Comment (jecfish) [reply site/en/docs/devtools/open/index.md]: Ooops, it was not. Fixed.

**Comment by jecfish** [comment]
Fixed. Please merge when it's approved.

**Review by sofiayem — APPROVED** [review]

**Pull request merged**